elements/rh-tabs/rh-tab.css

```diff
@@ -104,64 +122,32 @@ button:after {
 }
 
 /* Context switch on attributes */
-:host([active]) #container,
-#container.active {
+#button.active {
```

Comment:
The [APG recommends](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-automatic/) using [`aria-selected="true"` and `aria-selected="false"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected) to communicate to AT which tab is active. That said, that change might be out of scope for this PR.

Reply:
This definitely was a regression, good catch!  Resolved in https://github.com/RedHat-UX/red-hat-design-system/pull/1603/commits/87e038b5210301ad01012f0a568350d315909831

Reply:
Can you add an ax tree test for that using a11ySnapshot?